CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
-## How to contribute to the this project
+# How to contribute to this project
 
-Please use our tracker for new relation requests, etc
+Please use our tracker for requests for new relations, modifications, clarifications, etc.
 
 https://github.com/oborel/obo-relations
 
-## Contributing via Pull Requests
+Creating a new issue in the tracker allows the RO editors and community to discuss proposed changes, fixes, or additions. If you'd like faster turnaround on your request, and you know how to use GitHub, please specify your desired changes by submitting a GitHub Pull Request on the appropriate file.
 
-We hope to be able to accept contributions to the ontology in the form of Pull Requests in the future. Note that this is currently difficult for OWL, see:
+## Before you write a new request, please consider the following: 
 
-http://douroucouli.wordpress.com/2014/03/30/the-perils-of-managing-owl-in-a-version-control-system/
+- **Does the term already exist?** Before submitting suggestions for new ontology terms, check whether the term exist, either as a primary term or a synonym term. You can search using [OLS](http://www.ebi.ac.uk/ols/ontologies/ro)
```

Comment:
Minor typos:
- `exist` vs. `exists`
- period after OLS link

Reply:
well spotted. i copied this text from elsewhere but didn't notice those minor typos.